infra/terraform/modules/data_access/common/sentry/sentry/sentry.py

```diff
@@ -0,0 +1,26 @@
+'''
+Environment variables:
+ - ENV, e.g. "dev", "alpha", etc...
+ - SENTRY_DSN, Sentry DSN
+'''
+
+import os
+
+from raven import Client as Sentry
+from raven.transport.http import HTTPTransport as SentryHTTPTransport
+
+
+def catch_exceptions(fn):
+    def wrapped(*args, **kwargs):
+        try:
+            fn(*args, **kwargs)
+        except Exception:
+            client = Sentry(
+                os.environ["SENTRY_DSN"],
+                transport=SentryHTTPTransport,
+                environment=os.environ["ENV"],
+            )
+            client.captureException()
+            raise
+
+    return wrapped
```

Comment:
I'll move the code in the `__init__.py`s, although one benefit of it is that you can explicitly decide what to expose to the client code and what not. But for now probably these are simple enough that code can happily stay all in the `__init__.py`s files.